schemas-extractor/template/generate-schema.go

```diff
@@ -127,8 +126,11 @@ func export(v *schema.Schema) SchemaDefinition {
 	}
 
 	// TODO: Find better solution
-	if defValue, err := v.DefaultValue(); err == nil && defValue != nil && !reflect.DeepEqual(defValue, v.Default) {
```

Comment:
I couldn't see what `DeepEqual` was achieving here, perhaps something important that I misunderstood?

Reply:
Can't remember why I've added it here. Does removing changes anything?

Reply:
Using `!reflect.DeepEqual(...` there are 265 defaults generated.  Removing it makes 3134 defaults generated.    The diff is too big to compare everything but I think it is correctly finding all default values when removed.